src/Services/SpreadSheetService.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+/**
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ * @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @author  Stephen Waite <stephen.waite@cmsvt.com>
+ * @copyright Copyright (c) 2021-2022  Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @copyright Copyright (c) 2022 Stephen Waite <stephen.waite@cmsvt.com>
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ *
+ * This middleware is to allow a uniform use of the spreadsheet library.
+ * By passing an array values, a spreadsheet can be dropped to the client from anywhere
+ * To use this service send the file name and an array like this
+ * The first row is the headers that can be dynamcally generated or static
```

Comment:
does this comment need to be modified since looks like fields parameter is heading and doesn't like like there is row labels (the Q1, etc stuff)?